extras/drift_devtools_extension/README.md

```diff
@@ -5,6 +5,37 @@ based on the [drift_db_viewer](https://pub.dev/packages/drift_db_viewer) package
 
 To run the extension in standalone mode, run
 
-```
+```console
 flutter run -d chrome --dart-define=use_simulated_environment=true
 ```
+
+You'd also need to run an app with a drift database for you to be able to see it running.
+You can use our example app for that.
+
+```console
+cd ../../examples/app/
+flutter run --debug
+```
+
+Once the example app launches, it will print out something like:
```

Comment:
For me it doesn't print a ws URI, I wonder if that's OS specific? Mine says:

```
A Dart VM Service on Linux is available at: http://127.0.0.1:42439/XZyfqXFUYlg=/
The Flutter DevTools debugger and profiler on Linux is available at:
http://127.0.0.1:9102?uri=http://127.0.0.1:42439/XZyfqXFUYlg=/
```

I can paste the first link into the simulated environment without any modifications.

Reply:
I think you are running from the CLI, right? I remember it being different somehow. In vsCode at least it prints as I wrote. I could test it on Android Studio as well to see if this is a debugger/LSP thing or what. Anyway, I can rewrite this section when I do that.

Reply:
I asked DanTup (developer of the Dart-Code extension) on Discord and I got the answer:

> I'm not sure exactly who is printing each of those lines, but the first line in the first example is printed by the debug adapter - it prints the VM Service URL that it is connecting to.
> 
> In the second example, the URL is essentially the same, but it looks like it is really the old "Observatory URL", which was a web app used for debugging. You'd access that directly in a browser, to it uses HTTP as the protocol, and doesn't have the "/ws" websocket suffix.
> 
> Many Dart tools will accept the second format (a HTTP "observatory URL") and just change the protocol and add /ws automatically.
> 
> The final URL in the second example is that dart run has automatically started a DevTools server and is providing the URL. When running from VS Code, Dart-Code will do this internally and doesn't print the URL because you can access DevTools using the built-in commands instead (whereas from the CLI, printing the URL is the only way to give you it).
> 
> (and finally, the reason the port numbers and the strings after the port number are different between each run is just that they're random... a random port is used to avoid clashing with other instances that might be running and the other string is a random authentication token to ensure no other processes that can connect to your localhost can scan for VM Services and use them to evaluate code)

Reply:
I'll rewrite this part and commit it here.

Reply:
> I think you are running from the CLI, right?

Well your previous snippet suggested running the app on the CLI :D Perhaps we should mention both?

Reply:
I realized that hahaha. Did mentioned the output for both.